owncloud/indico_owncloud/client/index.js

```diff
@@ -0,0 +1,17 @@
+// This file is part of the Indico plugins.
+// Copyright (C) 2014 - 2021 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+
+window.setupOwncloudFilePickerWidget = function setupOwncloudFilePickerWidget({
+  filepicker_url,
+  fieldId,
+}) {
+  window.addEventListener('message', function(message) {
+    if (message.origin === filepicker_url) {
```

Comment:
Will this trigger for other cases such as browser extensions sending messages? Best to be as specific as possible to make sure only your own messages are handled... (e.g. also by sanity-checking the received data)

Reply:
Added checks to make sure the source is the file-picker's iframe and the incoming message has `files` key.